### ISSUE TITLE
[BUG] the download doesn't work

### ISSUE THREAD

**Issue by joaoluiz456a** [hydralauncher/hydra#969]
### Describe the bug

The download starts but it doesn't increase in speed. It just drops infinitely to zero and stays like that. I already uninstalled it, reinstalled it, but it doesn't work, I finished the task in the task manager. I didn't find a solution anywhere. I hope you help me, thank you. I did it through Google Translate so the English must be horrible. 
is the last version released

### Steps to Reproduce

I was just going to download normally and it gave the error

### Expected behavior

I expected it to install normally but this error occurs

### Screenshots

![imagem_2024-09-15_145815587](https://github.com/user-attachments/assets/42df204f-076f-4e11-abd3-256974d1dcfd)
![imagem_2024-09-15_145833881](https://github.com/user-attachments/assets/13496d76-1a64-4c9f-8415-00ea6118a43c)
![imagem_2024-09-15_145856625](https://github.com/user-attachments/assets/ba286db3-ce69-4c07-bc41-acd5da15bedb)

### Operating System

Windows 10

### Hydra Version

v2.1.3 "Leviticus"

### Before opening this Issue

- [X] I have searched the issues of this repository and believe that this is not a duplicate.
- [X] I am aware that Hydra team does not offer any support or help regarding the downloaded games.

**Comment by xjhowxjhow** [comment]
@joaoluiz456a  olá troque de fonte, talvez esta sem peers

![image](https://github.com/user-attachments/assets/ff6ecd57-d584-410c-ae46-04eb85d44bee)

**Comment by Maguz0** [comment]
siguen sin aparecer las opciones de descarga en esta version nueva v2.1.3 "Levítico" podrian arreglarlo 
![Uploading Hydra 15 sep. 2024 12_54_52.png…]()

**Comment by joaoluiz456a** [comment]
mano, mas eu estou com as fontes normais do hydralink era ter peers né ? pq estou na fonte desse site que eles recomendam https://hydralinks.cloud/  e todas elas não funcionam

**Comment by zamitto** [comment]
> mano, mas eu estou com as fontes normais do hydralink era ter peers né ? pq estou na fonte desse site que eles recomendam https://hydralinks.cloud/ e todas elas não funcionam. tem alguma fonte que recomenda? ele inicia com peers mas ele fica com 0 logo em seguida

Could you upload your logs file? 
They are located at "%appdata%/hydra"

**Comment by joaoluiz456a** [comment]
yes, i can
[error.txt](https://github.com/user-attachments/files/17035229/error.txt)
[info.txt](https://github.com/user-attachments/files/17035230/info.txt)
[logs.txt](https://github.com/user-attachments/files/17035231/logs.txt)
[pythoninstance.txt](https://github.com/user-attachments/files/17035232/pythoninstance.txt)

**Comment by zamitto** [comment]
@joaoluiz456a, could find any strange log in your files.
Every game that your tried failed to download?

Could your follow a few steps?
1) Delete the 4 logs files
2) Restart Hydra
3) Try to download some games that you want
4) Reupload your files here

**Comment by Maguz0** [comment]
sigue igual en la version v2.1.4 no aparece las opciones de descarga

**Comment by joaoluiz456a** [comment]
yes,  I deleted them and and gave this
![imagem_2024-09-17_191440162](https://github.com/user-attachments/assets/cb620a73-6100-4b7b-af8c-0e09d9a1b70d)
[error.txt](https://github.com/user-attachments/files/17035601/error.txt)
[info.txt](https://github.com/user-attachments/files/17035606/info.txt)
[pythoninstance.txt](https://github.com/user-attachments/files/17035607/pythoninstance.txt)

he created these three I think the error is there, there is some resolution from what I understand it is in the url but, as I said, I use the url that everyone uses

**Comment by zamitto** [comment]
@joaoluiz456a yeah, nothing in these logs indicate what might be happening to you. Any chance the download directory is protected by Windows or something?

**Comment by joaoluiz456a** [comment]
I really wouldn't know how to look if there is something wrong in the directory but I don't think it's the directory, because before it worked when finishing the task in the task manager, but today it doesn't work. When it didn't work before I uninstalled Hydra and it worked but now it doesn't work anymore. I have no ideas on how to resolve this.

**Comment by iurisouz4** [comment]
@joaoluiz456a having same issue here, resolved closing Docker Desktop and stopping a few services running in port 3333 and 3000, and it just worked... maybe it was a port conflict, i'll try to isolate the problem later

**Comment by joaoluiz456a** [comment]
How do you do what you said? I have no idea/ como faz isso que voce disse? eu não tenho ideia

**Comment by JackEnx** [comment]
@joaoluiz456a could you check if the problem persists in 3.0?
